crates/nu-command/tests/commands/open.rs

```diff
@@ -260,3 +260,21 @@ fn open_dir_is_ls() {
         assert_eq!(actual.out, "3");
     })
 }
+
+
+#[test]
+fn test_open_block_command() {
+    let actual = nu!(
+        cwd: "tests/fixtures/formats",
+        r#"
+            def "from ssv" [] { lines | split column ";" }
```

Comment:
`from ssv` is already a command inside of nushell. Just to make the test clearer, you might want to choose one that is specific to the test and not part of nushell

Reply:
Oh, I wasn't aware that ssv actually exists.  I've changed it and I'm pretty sure the new custom file extension 'blockcommandparser' with ',|,' as separator doesn´t exist.

And sorry, I forgot to run the checks, especially 'cargo fmt' on my last commit.